### ISSUE TITLE
Quicklook preview on Sequoia

### ISSUE THREAD

**Issue by AlexPerathoner** [vincentneo/Avenue-GPX-Viewer#43]
Since updating to Sequoia, the map in the quicklook preview isn't being loaded:
![immagine](https://github.com/user-attachments/assets/18f31a4c-acd9-489e-8664-c3d5773078b7)

This happens with every gpx I've tried, regarless of the file location (on iCloud or locally), if the firewall is turned on or off.

**Comment by vincentneo** [comment]
Yes, I am aware of the issue but unsure of why macOS Sequoia breaks it. 

I have already filed for code level support from Apple to help with this, though I have yet to hear back yet.

**Comment by AlexPerathoner** [comment]
Cool, nice to see it wasn't a weird setting I had perhaps in some firewall. Thanks for being already working on it

**Comment by vincentneo** [comment]
By the way, since developer support wants the issue posted on their forum platform, if you are interested, you can check out: https://developer.apple.com/forums/thread/766615 for the latest (technical) updates (nothing right now)

**Comment by pvdb** [comment]
I have noticed the same, so just commenting on this issue to receive update notifications :sweat_smile:

**Comment by adambene** [comment]
Any updates or plan for the missing map in quick look?

**Comment by AlexPerathoner** [comment]
@pvdb 

<img width="330" height="206" alt="Image" src="https://github.com/user-attachments/assets/70474a8e-bb18-4109-a8b4-fdcd46aceb76" />

FYI there's also a Notification Tab on the side, the status option is really nice

**Comment by vincentneo** [comment]
Hi all, please give Avenue 1.6.1 a try, which should fix this, albeit with a workaround. I would like to receive feedback on whether if it is working for all of you. 

https://github.com/vincentneo/Avenue-GPX-Viewer/releases/tag/1.6.1

This version will not be available on the App Store due to reasons detailed in the release page. Thank you!

**Comment by AlexPerathoner** [comment]
Hi, I love you. Works perfectly. Hope the sponsor eases you a bit from having had to deal with the Apple Engineers Support :)

**Comment by vincentneo** [comment]
Appreciate it, thank you!
